content/modules/modules-quickstart.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Modules Quick Start
+description:
+weight: 1
+alwaysopen: false
+categories: ["Modules"]
+---
+To quickly get access to a Redis Enterprise database with a module,
+you can sign up for a free Redis Cloud database and enable the module that you want to try out.
+
+The modules that are currently available are:
+
+- [RediSearch]({{< relref "/modules/redisearch/_index.md" >}})
+- [RedisGraph]({{< relref "/modules/redisgraph/_index.md" >}})
+- [RedisBloom]({{< relref "/modules/redisbloom/_index.md" >}})
+- [RedisJSON]({{< relref "/modules/redisjson/_index.md" >}})
+- [RedisTimeSeries]({{< relref "/modules/redistimeseries/_index.md" >}})
+
+The steps for getting up and running with a database that has a module enabled are:
+
+1. Create a new subscription.
+1. Create a new database with the module enabled.
+1. Connect to the database.
+
+## Step 1: Create a New Subscription
+
+To create a new subscription:
+
+1. In the Redis Cloud menu, click **Subscriptions**.
+1. At the bottom of the page, click ![Add](/images/rs/icon_add.png#no-click "Add").
+1. Select your subscription configuration:
+    1. Select a cloud provider: **Amazon AWS**, **Microsoft Azure**, **Google Cloud Platform**
```

Comment:
This needs to be revised.  It's currently only available on AWS and in Mumbai region (next line)

Reply:
@we should add that we'll have more regions soon.  If necessary we can say this is now in `Public Preview`